### PR TITLE
UwU-ifying the whole bot

### PR DESCRIPTION
**Pull request by Eliiia**
(and changing some phrasing of permission stuffz)

### CODE REVIEW

File: src/commands/perms.js [hunk 1]
```diff
@@ -9,10 +9,10 @@ const permissions = [`CREATE_INSTANT_INVITE`, `KICK_MEMBERS`, `BAN_MEMBERS`, `MO
 `VIEW_GUILD_INSIGHTS`, `READ_MESSAGE_HISTORY`, `SEND_MESSAGES`, `SEND_TTS_MESSAGES`, `ATTACH_FILES`, `USE_EXTERNAL_STICKERS`, `ADD_REACTIONS`, `SEND_MESSAGES_IN_THREADS`,
 `CREATE_PUBLIC_THREADS`, `CONNECT`, `SPEAK`, `PRIORITY_SPEAKER`, `STREAM`, `MUTE_MEMBERS`, `DEAFEN_MEMBERS`, `MOVE_MEMBERS`, `MENTION_EVERYONE`];
 
-let permsOutput = `\`\`\`\nServer - 📛\nCurrent channel - ♨️\n\n📛 | ♨️\n`;
+let permsOutput = `\`\`\`\nUser permissions in:\n  Server: 📛\n  Current channel: ♨️\n\n📛 | ♨️\n`;
```
Comment (Agerr) [inline src/commands/perms.js:12]: Remove spaces after \n's

File: src/modules/botPerms.js [hunk 2]
```diff
@@ -12,23 +12,18 @@ module.exports = async (message, info) => {
         const yesEmote = `✅`;
         const noEmote = `⛔`;
 
-        let permsOutput = `
-            Server - 📛
-            Current channel - ♨️
-
-            📛 | ♨️
-
+        let permsOutput = `User permissions in:\n  Server: 📛\n  Current channel: ♨️\n\n📛 | ♨️
```
Comment (Agerr) [inline src/modules/botPerms.js:15]: Please remove spaces after \n's....

Comment (Agerr) [inline src/modules/botPerms.js:15]: Also its Botperms not user permissions smh lol

**Review by Agerr — CHANGES_REQUESTED** [review]
A few things to change:

- Check comments i wrote out whats wrong
- Please change all config.emojis to for format of **FirstSecondThird** words rather than firstsecondthird.

File: src/commands/perms.js (hunk 1 above)
Comment (Eliiia) [reply src/commands/perms.js:12]: makes it a lot clearer but ok done

Comment (Agerr) [reply src/commands/perms.js:12]: @Eliiia No it messes up how it looks on mobile im pretty sure

File: src/commands/dm.js [hunk 3]
```diff
@@ -7,27 +7,27 @@ module.exports.run = async (bot, message, args) => {
 
     let user = await bot.users.fetch(args[1]).catch(error => {
         success = false;
-        message.channel.send({ content: `Error while fetching user.\`\`\`${error}\`\`\`` });
+        message.channel.send({ content: `Error!!! <${config.emojis.pandaScared}>\`\`\`${error}\`\`\`` });
```
Comment (Agerr) [inline src/commands/dm.js:10]: Still pandaScared -> PandaScared

File: src/commands/slowmode.js [hunk 4]
```diff
@@ -1,20 +1,22 @@
 const timeConvert = require(`../modules/timeConvert.js`);
 
 module.exports.run = async (bot, message, args) => {
-    if (!message.member.permissions.has(`MANAGE_CHANNELS`)) return message.channel.send({ content: `The \`${args[0]}\` command requires "Manage channels" permission. `});
+    if (!message.member.permissions.has(`MANAGE_CHANNELS`)) return message.channel.send({ content: `The \`${args[0]}\` command requires "Manage Channels" permission, which I don't have :c`});
```
Comment (Agerr) [inline src/commands/slowmode.js:4]: Remove that "which i dont have part", its about user not bot

File: src/modules/fetchMember.js [hunk 5]
```diff
@@ -24,8 +24,8 @@ module.exports = async (message, memberInfo) => {
 
                     embed = new Discord.MessageEmbed()
                         .setColor(config.color)
-                        .setTitle(`Multiple users share this name:`)
-                        .setDescription(`${usersList}\nTry using @mention, id or name#tag`);
+                        .setTitle(`Multiple users have this name!! <${config.emojis.pandaScared}>`)
```
Comment (Agerr) [inline src/modules/fetchMember.js:27]: pehdbnasvflk;./asfda

**Review by Agerr — CHANGES_REQUESTED** [review]

File: src/modules/fetchMember.js (hunk 5 above)
Comment (Eliiia) [reply src/modules/fetchMember.js:27]: what

**Review by Agerr — CHANGES_REQUESTED** [review]
Change message of permissions "xx" required equally, either both to same or none to none

**Review by Agerr — CHANGES_REQUESTED** [review]
Eli,,,,,,,,, more changes.... are required........

**Pull request merged**